blocks/accordion/accordion.css

```diff
@@ -0,0 +1,62 @@
+.accordion details {
+  border-bottom: 2px solid #ccc;
+}
+
+/* stylelint-disable-next-line no-descending-specificity */
+.accordion details + details {
+  margin-top: 14px;
+}
+
+.accordion details p {
+  margin-bottom: .8em;
+  margin-top: .8em;
+}
+
+.accordion details summary {
+  position: relative;
+  padding: 0 16px;
+  cursor: pointer;
+  list-style: none;
+  overflow: auto;
+  font-family: Solway, serif;
+  font-size: 20px;
+  color: #093567;
+}
+
+.accordion details[open] summary {
+  background-color: var(--transparent);
+}
+
+.accordion details summary::-webkit-details-marker {
+  display: none;
+}
+
+.accordion details summary::after {
+  content: '+';
+  position: absolute;
+  font-size: 25px;
+  top: 10%;
+  right: 22px;
+  transition: transform 0.4s;
+}
+
+.accordion details[open] summary::after {
+  transform: rotate(-180deg);
+  content: '-';
+}
+
+.accordion details .accordion-item-body {
+  padding: 0 20px;
+  max-height: 0;
+  overflow: hidden;
+  transition: max-height 0.4s ease;
```

Comment:
I'm having trouble with animating the expansion of the accordion when you click on one of the elements. These lines are what I'm expecting would work, but isn't working for me.
Any pointers @helms-charity @kailasnadh790?

Example from original site: https://www.learninga-z.com/site/contact/support

Reply:
@silviu also we need to see if we can fix the CLS on mobile.

Reply:
Fixed CLS on mobile